src/codegen_call.cpp

```
@@ -787,6 +779,31 @@ CodeGen::MapFields CodeGen::loadMapHeader(llvm::Value *mapVal, const std::string
     return f;
 }
 
+// ===== Header stores =====
+
+void CodeGen::storeListHeaderFields(llvm::Value *headerPtr, llvm::Value *len,
```

Clang (14.0.6)

[warning] 4 adjacent parameters of 'storeListHeaderFields' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)